Auth/OpenID/Parse.php

```diff
@@ -218,12 +218,12 @@ function removeQuotes($str)
     
     function match($regexp, $text, &$match)
     {
-        if (!is_callable('mb_ereg_search_init')) {
-            if (!preg_match($regexp, $text, $match)) {
-                return false;
-            }
-            $match = $match[0];
-            return true;
+        if (preg_match($regexp, $text, $match)) {
+           return true;
+        }
+
+        if ( !is_callable('mb_ereg_search_init')) {
```

Comment:
There's no need for this block now.  There was never a need for it since preg_match replaced mb_ereg_search a while ago. Just delete this block and replace it with `return false;`